src/main/app/controller/GetController.ts

```diff
@@ -7,23 +7,28 @@ import { AppRequest } from './AppRequest';
 
 @autobind
 export class GetController {
-  constructor(protected readonly name: string, protected readonly content: Record<string, Record<string, unknown>>) {}
+  constructor(
+    protected readonly view: string,
+    protected readonly content: Record<'en' | 'cy' | 'common', Record<'divorce' | 'civil' | string, unknown>>
+  ) {}
 
   public async get(req: AppRequest, res: Response): Promise<void> {
     if (res.locals.isError) {
       // If there's an async error, it wil have already rendered an error page upstream,
       // so we don't want to call render again
       return;
     }
+    const language = req.session?.lang || 'en';
+    const serviceType = res.locals?.serviceType || 'divorce';
 
-    const languageContent = this.content[req.session.lang] || this.content['en'] || {};
-    const commonLanguageContent = commonContent[req.session.lang] || commonContent['en'];
+    const languageContent = this.content[language]?.[serviceType] || {};
```

Comment:
we don't need that default any more?

Reply:
👍 Good spot, I've removed this in 9b52937.